src/main/java/de/tum/in/www1/artemis/repository/ResultRepository.java

```diff
@@ -702,8 +702,8 @@ default Result findFirstWithFeedbacksByParticipationIdOrderByCompletionDateDescE
                 .orElseThrow(() -> new EntityNotFoundException("Result by participationId", participationId));
     }
 
-    default Result findWithEagerSubmissionAndFeedbackAndAssessorByIdElseThrow(long resultId) {
-        return findWithEagerSubmissionAndFeedbackAndAssessorById(resultId).orElseThrow(() -> new EntityNotFoundException("Result", resultId));
+    default Result findWithEagerSubmissionAndFeedbackAndAssessorAssessmentNoteByIdElseThrow(long resultId) {
```

Comment:
Should be `findWithEagerSubmissionAndFeedbackAndAssessorAndAssessmentNoteByIdElseThrow` (`And` missing).

Reply:
Thanks a lot.